samples/net/sockets/echo_server/overlay-vpn.conf

```diff
@@ -0,0 +1,8 @@
+CONFIG_WIREGUARD=y
+CONFIG_WIREGUARD_MAX_SRC_IPS=4
+
```

Comment:
this really seems to want some more IP and stack, should these have some combo of

```
CONFIG_NET_IF_UNICAST_IPV6_ADDR_COUNT=4
CONFIG_NET_IF_UNICAST_IPV4_ADDR_COUNT=2
CONFIG_NET_IF_MAX_IPV6_COUNT=4
CONFIG_NET_IF_MAX_IPV4_COUNT=4

CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=2048
CONFIG_NET_RX_STACK_SIZE=2048
```
?

Reply:
Yes noticed that too, I will add relevant options there.